coding-challenges/faq-component/styles.css

```
@@ -0,0 +1,44 @@
+body {
+    font-family: Arial, sans-serif;
+    background-color: #e0d9d9;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+}
+
+.faq-main-container {
+    background: white;
+    padding: 20px;
+    border-radius: 10px;
+    width: 400px;
+    box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.1);
+}
+
+h1 {
+    font-size: 24px;
```

we should avoid pixels for font sizes.

```suggestion
    font-size: 1.5rem;
```